sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 1.1.0-beta.4 (2022-04-11)
+
+### Features Added
```

Comment:
Delete Features Added.

Should look like
```
### Other Changes

#### Dependency Updates
- Upgraded `azure-communication-common` to 1.1.2
- Upgraded `azure-core` to 1.27.0
- Upgraded `azure-identity` to 1.5.0
```

Reply:
Done.